src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,35 @@
+import { useEffect, useState } from 'react';
+import { getPeople } from '../../api';
+import { PeopleTable } from './PeopleTable';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState([]);
+  const [error, setError] = useState('');
+
+  useEffect(() => {
+    async function result() {
+      try {
+        const peopleFromServer = await getPeople();
+
+        setPeople(peopleFromServer);
+      } catch {
+        setError('Cannot load people');
+      }
+    }
```

Comment:
Find a mother and a father by motherName and fatherName and add them to the person for future use